components/dashboard/src/admin/BlockedRepositories.tsx

```diff
@@ -15,6 +15,7 @@ import CheckBox from "../components/CheckBox";
 import { ItemFieldContextMenu } from "../components/ItemsList";
 import { ContextMenuEntry } from "../components/ContextMenu";
 import Alert from "../components/Alert";
+import { SpinnerLoader } from "../components/Loader";
```

Comment:
**issue**: The SVG added seems cropped, although it renders ok. Is this using the code from https://github.com/gitpod-io/gitpod/issues/15896#issuecomment-1400646225? I'm adding below one more version with a square bounding box.

```svg
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 16 16">
  <g clip-path="url(#a)">
    <path fill="#A8A29E" fill-rule="evenodd" d="M11.4284.944629C10.2681.324456 8.97283 0 7.65722 0v3c2.76138 0 4.99998 2.23858 4.99998 5 0 2.7614-2.2386 5-4.99998 5-1.38071 0-2.63071-.5596-3.53553-1.4645l-2.12132 2.1214c.93027.9302 2.07561 1.6167 3.33457 1.9986 1.25896.3819 2.59267.4474 3.883.1908 1.29036-.2567 2.49746-.8276 3.51446-1.6622 1.0169-.8346 1.8124-1.9072 2.3159-3.1226.5034-1.2155.6994-2.53636.5704-3.84564-.129-1.30927-.5788-2.56653-1.3097-3.66042-.7309-1.09389-1.7203-1.99064-2.8806-2.610811Z" clip-rule="evenodd"/>
  </g>
  <defs>
    <clipPath id="a">
      <path fill="#fff" d="M0 0h16v16H0z"/>
    </clipPath>
  </defs>
</svg>
```